ta/crypt/cryp_taf.c

```diff
@@ -7,7 +7,9 @@
 
 #include <assert.h>
 #include <cryp_taf.h>
+#if defined(CFG_MEMTAG)
 #include <memtag.h>
```

Comment:
I am not sure why this needs to be included - none of the code in this file calls any `memtag_*` calls.

Reply:
Then just remove it.